### ISSUE TITLE
[A11y]Syntax highlighting in Read Mes does not have sufficient contrast.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9269]
Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org/packages/AutoMapper/
Windows version: Windows 10
ChromiumEdge Version: 105.0.1343.42 (Official Build) (64-bit)

Checking the URL above, the syntax highlighting in the readme does not meet sufficient contrast requirements.

**Comment by joelverhagen** [comment]
Ideas:
1. Play wack-a-mole with reporting contrast issues and override rules manually
    - [ ] Can get ahead of the reports with Accessibility Insights FastPass (to proactively scan top packages w/ readmes)
3. Use an accessible stylesheet provided by highlight.js (looks different)
